weareinreach/inreach-api
# Feature - Database/Migration - add and remove 'community-asylum-seekers' service and organizations property

## Description
This database migration uses two files: 

services-keep-property.json
services-remove-property.json

to remove and add the 'community-asylum-seeker' property to select services, and remove the 'community-asylum-seeker- organization property from organizations that have no services with the service property

## Asana ticket:
https://app.asana.com/0/1202317426899416/1203051703623853
## PR Checklist




## How to Test
the migration should alter the same number of lines as the rollback, also queries can be made in mongo db to observe the changes after each migration/rollback is done

note: the rollback requires files that are written from the migration, so if the rollback is run without a prior migration, there will be errors
